backend/database/seeds/CardsTableSeeder.php

```diff
@@ -11,6 +11,14 @@ class CardsTableSeeder extends Seeder
      */
     public function run()
     {
-        factory(App\Models\Card::class, 10)->create();
+        $faker = \Faker\Factory::create();
+
+        DB::table('attribute_card')->insert(
```

Comment:
Does this only create 1 card, 1 attribute, and link them?

What do you think about creating a factory to create attributes, and a separate factory to create cards (which is called after, and assigns each card multiple attributes)?

Reply:
yeah, do you want to do that in a seperate PR? 😄